docs/pages/demo/daterangepicker/index.mdx

```diff
@@ -13,9 +13,14 @@ import * as CustomRangeInputs from './CustomRangeInputs.example';
 
 <PageMeta component="DateRangePicker" />
 
-## Date picker
+## Date range picker
 
-[Date pickers](https://material.io/components/pickers/) let users select a date, or a range of dates. They should be suitable for the context in which they appear.
+> ⚠️⚠️ The date range picker is unstable, it's **not suitable** for usage in production. ⚠️⚠️
+
+> The date range picker will be made available in the coming month for production usage as part of a paid extension of the community edition (MIT license) of Material-UI.
+> This paid extension will include advanced components (rich data grid, date range picker, tree view drag & drop, etc.). Pricing plans will start at \$129/developer for early access.
+
+The [date range pickers](https://material.io/components/pickers/) let users select a range of dates. They should be suitable for the context in which they appear.
```

Comment:
> They should be suitable for the context in which they appear.

For example?

Reply:
So ?
```suggestion
The [date range pickers](https://material.io/components/pickers/) let users select a range of dates. 
```

Reply:
I was asking what, specifically, "They should be suitable for the context in which they appear." referred to? Suitable in what sense? What "context"? But If the sentence isn't needed, then yes it can be deleted.